packages/networks/src/networks/zksync.ts

```diff
@@ -0,0 +1,24 @@
+import { NetworkConfig } from '@relay-protocol/types'
+
+export const zksync: NetworkConfig = {
+  l1ChainId: 1,
+  chainId: 324,
+  isTestnet: false,
+  name: 'Zksync',
+  slug: 'zksync',
+  hyperlaneMailbox: '0xf44AdA86a1f765A938d404699B8070Dd47bD2431',
+  // no CCTP on zksync https://developers.circle.com/stablecoins/evm-smart-contracts
```

Comment:
Great! Does that mean USDC on zksync is just a "bridged" token?

Reply:
yes it used the zksync standard ERC20 bridge